packages/wallets/src/features/cashier/modules/CashierLocked/CashierLocked.tsx

```diff
@@ -65,6 +65,10 @@ const CashierLocked: React.FC<TCashierLockedProps> = ({ children, module }) => {
         poiNeedsVerification,
     });
 
+    if (!status) {
+        return <Loader />;
+    }
```

Comment:
Changed condition to check if data is returned instead of using isLoading to cater for scenario where data is already cached and no loading exists:

https://github.com/binary-com/deriv-app/assets/142860499/98e344d4-5c15-4dd3-952d-c86254460cad